idearium-lib/lib/query.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+/**
+ * Assign some defaults to the passed Mongoose query options.
+ * @param {Object} options Mongoose query options.
```

Comment:
Could you please use the [jsdoc approach](http://usejsdoc.org/tags-param.html#parameters-with-properties) to documenting options objects...